FhirToDataLake/NuGet.Config

```diff
@@ -2,6 +2,7 @@
 <configuration>
   <packageSources>
     <clear />
+    <add key="nuget.org" value="https://api.nuget.org/v3/index.json" />
```

Comment:
Is this a backup resource for our Nuget packages? Maybe put it after our default resource so that our provided packages have higher priority.